frontend/src/components/Contexts/LanguageContext.tsx

```diff
@@ -57,7 +57,7 @@ export const LanguageProvider: FC<Props> = ({ children }) => {
 
 export const useLanguageContext = () => useContext(LanguageContext)
 
-export const TranslateText = (id: string): string => {
+export const TranslateTextUnsafe = (id: string): string => {
```

Comment:
Don't call it unsafe, this is open source, call is deprecated or old or something

Reply:
I was thinking of calling it unsafe since it can cause frontend exceptions if used incorrectly. I don't think this poses a security risk since it is unrelated to the backend. I'm a little hesitant to calling it deprecated since we do need to use it in a few places where we want to translate outside of React components. It's not necessairly a direct replacement, but it can be good to have the name reflect that it is not the ideal function to use. But as long as this is clear I can change it to something else like "TranslateTextWithContext"

Reply:
I think that's better, change the name and then you can merge